### PR TITLE
Use launcher.bat instead of a go binary for windows

### PR DESCRIPTION
**Pull request by mceachen**
The launcher binary written in go was frequently marked as a trojan or virus. A simple bat file should seen as more innocuous. See https://github.com/electron/spectron/issues/93 for the problem as well as this bat file.

Proof that the bat file will be marked as safe: https://www.virustotal.com/en/file/1d86bd7934a6628d34458708ae435d4decefb671d51e262ad1f8d7f66ae89542/analysis/1495475045/

Proof that launcher.exe is marked as malicious: https://www.virustotal.com/en/file/dee7112c53b6bd2509ec23c5c6b4f17c65210684789510fa4caf6d3ff4c66949/analysis/

### CODE REVIEW

File: lib/application.js [hunk 1]
```diff
@@ -150,12 +150,8 @@ Application.prototype.createClient = function () {
       args.push(arg)
     })
 
-    var launcherPath = null
-    if (process.platform === 'win32') {
-      launcherPath = path.join(__dirname, '..', 'bin', 'launcher.exe')
-    } else {
-      launcherPath = path.join(__dirname, 'launcher.js')
-    }
+    const isWin = (process.platform === 'win32')
```
Comment (kevinsawicki) [inline lib/application.js]: Parens not needed here

Comment (kevinsawicki) [inline lib/application.js]: I'd prefer not to use `const` in spectron yet to not break any old node versions that people may still be using.

File: lib/application.js [hunk 2]
```diff
@@ -150,12 +150,8 @@ Application.prototype.createClient = function () {
...
+    const launcherPath = path.join(__dirname, isWin ? 'launcher.bat' : 'launcher.js')
```
Comment (kevinsawicki) [inline lib/application.js]: Please use `var` instead of `const`

Comment (mceachen) [inline lib/application.js]: done

File: lib/application.js (hunk 1 above)
Comment (mceachen) [inline lib/application.js]: fair enough, done.

**Comment by mceachen** [comment]
The AppVeyor failures are both odd. The [32 bit build](https://ci.appveyor.com/project/kevinsawicki/spectron/build/job/4sq98q17e1nhj6xd) died in `npm install`, and the [64 bit build](https://ci.appveyor.com/project/kevinsawicki/spectron/build/job/4m67llk2sh5j4714) died from

```
  1) multiple windows launches the application:
      AssertionError: expected 'Bottom' to equal 'Top'
      + expected - actual
      -Bottom
      +Top
      
      at title() - node_modules\webdriverio\build\lib\commands\getTitle.js:31:28
      at execute(<Function>, "getBounds", , "require") - lib\api.js:295:19
Command exited with code 1
```

which isn't a failure I've seen before. Is that a known flaky test?

**Comment by kevinsawicki** [comment]
>  Is that a known flaky test?

Not that I've seen before, could be focus related.

**Pull request merged**